src/rez/build_system.py

```diff
@@ -301,6 +299,17 @@ def add_standard_build_actions(cls, executor, context, variant, build_type,
         for var, value in env_vars.items():
             executor.env[var] = value
 
+    @classmethod
+    def add_pre_build_commands(cls, executor, variant, build_type,
+                                   install, build_path, install_path=None):
+        """Perform build actions common to every build system.
+
+        This includes:
```

Comment:
This docstring is wrong, it describes what `add_standard_build_actions` used to do.